crates/oci-runtime-clap/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "oci-runtime-clap"
```

Comment:
So far we have used the lib prefix for our crates. I also would drop 'clap' in the name. I suggest libruntime-cli. @utam0k Thoughts?

Reply:
I agree.  I think so too. There may be cases like this one, where someone wants to create a cli but only using the youki backend, but let's think about that when the time comes.

Reply:
I like the idea of including "oci" in the title: as a component of the youki tree alone, it's redundant since the whole project is about OCI.  But I'm hoping this can become a crate published on its own, in which case putting OCI in the name makes its function clearer.  I'm not particularly attached to the current name though; names are hard.

How about `liboci-cli`?

Reply:
Both are great names! I think it should include `oci`.  So, let's go with `liboci-cli`

Reply:
Done!